src/FreeScribe.client/UI/SettingsWindow.py

```diff
@@ -145,6 +146,7 @@ def __init__(self):
             "singleline": False,
             "frmttriminc": False,
             "frmtrmblln": False,
+            "best_of": 6,
```

Comment:
2 likely a good number if we are putting one; default to not using this and can be enabled